PhysicsTools/HeppyCore/python/framework/heppy_loop.py

```diff
@@ -45,12 +45,14 @@ def runLoop( comp, outDir, config, options):
     fullName = '/'.join( [outDir, comp.name ] )
     # import pdb; pdb.set_trace()
     config.components = [comp]
+    memcheck = 2 if getattr(options,'memCheck',False) else 0
```

Comment:
Hi, 
If I understand correctly the logic of the Looper class, in order to have the mem checker being inactive by default this line should be like this:

memcheck = 2 if getattr(options,'memCheck',False) else -1
 
no ?